assets/js/actions/display.js

```diff
@@ -1,6 +1,14 @@
 export const UPDATE_DISPLAY_SETTING = 'UPDATE_DISPLAY_SETTING';
 
 export const updateDisplay = (desktopOnly) => {
+  if (desktopOnly) {
+    window.Intercom('update', { "hide_default_launcher": false });
+  } else {
+    if (window.innerWidth < 721) {
+      window.Intercom('update', { "hide_default_launcher": true });
+    }
```

Comment:
would there ever be a non-desktop instance where innerWidth > 721 that we would still want to hide?

Reply:
not for now, anything over 721 will always go to desktop mode

Reply:
do we still need the if then?

Reply:
its an edge case of:

Your browser window is small.
You switch to desktop only.
You make the screen size > 721.
You click the mobile mode button but your screen size still > 721.